automatic/peazip.install/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,17 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  file           = "$toolsDir\peazip-6.5.0.WINDOWS.exe"
+  file64         = "$toolsDir\peazip-6.5.0.WIN64.exe"
+  fileType       = 'exe'
+  packageName    = 'peazip.install'
+  softwareName   = 'PeaZip'
+  silentArgs     = '/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP-'
```

Comment:
Please change this line to
``silentArgs     = "/VERYSILENT /SUPPRESSMSGBOXES /NORESTART /SP- /LOG=`"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).Install.log`""``

It's a convention I'm trying to implement to get the installer log when installation fails